arches_lingo/src/arches_lingo/components/basic-search/SearchResult.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import type { Concept } from "@/arches_lingo/types.ts";
+import { getItemLabel } from "@/arches_lingo/utils.ts";
+
+defineProps({
+    searchResult: {
+        type: Object,
+        required: true,
+    },
+});
+
+const getParentLabels = (item: Concept, preferredLanguage: string): string => {
+    const arrowIcon = " → ";
+
+    return item.parents.reduce((acc, parent, index) => {
+        const label = getItemLabel(parent as Concept, preferredLanguage);
+        if (label) {
+            return acc + (index > 0 ? arrowIcon : "") + label;
+        }
+        return acc;
+    }, "");
+};
+</script>
+
+<template>
+    <div
+        class="search-result"
+        :class="{ 'is-even': searchResult.index % 2 === 0 }"
+    >
+        <i class="pi pi-paperclip" />
+
+        <div style="margin: 0 0.5rem">
+            {{ getItemLabel(searchResult.option, "en-US") }}
```

Comment:
why en-US instead of en? I think we un-americanized the test data.

Reply:
ah, this is just a placeholder til we get an endpoint that returns the user's language.